src/components/Common/OpenPositions.js

```diff
@@ -1,12 +1,10 @@
 import React from 'react'
+import { Padding } from 'styled-components-spacing'
 import { Col, Row } from '../grid'
 import JobsByLocation from '../JobsByLocation'
-import { Padding } from 'styled-components-spacing'
 import { SectionTitle, Subtitle } from '../Typography'
-import { Section } from './elements'
-import Hr from '../Common/Hr'
-import GetInTouch from '../Common/GetInTouch'
-import JobLink from '../Common/JobLink'
+import { Section } from '../JoinUs/elements'
```

Comment:
As we're now using this component in the homepage and joinus, we shouldn't rely on components defined in `JoinUs` as it increases obscurity and can cause issues in the future in that changing JoinUS page also affects the homepage.

Reply:
Looking more closely at this `Section`

```export const Section = ({ children, greyBg, ...props }) => {
  const content = <Grid {...props}>{children}</Grid>

  if (!greyBg) {
    return content
  }

  return <GreyBackground>{content}</GreyBackground>
}```

You can be sure that this section is going to be wrapped in a `GreyBackground` component at all times so lets get rid of the section here and just wrap the entire thing with `GreyBackground`